jekyll/_cci2/api-permissions.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: classic-docs
+title: "API Permissions"
+short-title: "API Permissions"
+description: "How to Add and Change API Permissions on CircleCI"
+order: 20
+---
+
+To use the CircleCI API
+or view details about a build,
+you will need API tokens with the appropriate permissions.
+This document describes the types of API tokens,
+as well as how to create and delete them.
+
+## Overview
+
+There are two types of API token
+you can create within CircleCI.
+
+  1. **Personal**:
+  These tokens are used to interact with the CircleCI API
+  and grant full read and write permissions.
+  2. **Project**:
+  These tokens provide information about specific projects
+  and only grant read permissions.
+  Project tokens have three scope options: _Status_, _Build Artifacts_, and _All_.
+      - _Status_ tokens grant read access to the project's build statuses.
+      Useful for [embedding status badges]({{ site.baseurl }}/2.0/status-badges/).
+      - _Build Artifacts_ tokens grant read access to the project's build artifacts.
+      - _All_ tokens grant read access for the project's build statuses and build artifacts.
+
+**Note**:
+Once created,
```

Comment:
Once-> After you create an API token (once is colloquial and could be confused for 'one time'

Reply:
[Added to Style Guide](https://github.com/circleci/circleci-docs/wiki/Style-Guide#once-upon-a-time)